api/v1/constraint-labels.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+type ConstraintLabels []Label
+
+type Label struct {
+	ID    string          `json:"id"`
+	Label string          `json:"label"`
+	Enum  map[string]uint `json:"enum"`
+}
+
+// handleGetConstraintLabels godoc
+//	@Summary		List all constraint labels for the tenant
+// 	@Param 		  token header string true "Auth token"
+//	@Produce		json
+//	@Success		200	{object}	ResponseEnvelope{response=ConstraintLabels}
+//	@Router			/constraint-labels [get]
+func (s *apiV1) handleGetConstraintLabels(c echo.Context) error {
+	return c.JSON(http.StatusNotImplemented, map[string]string{})
+}
+
+// TODO: PUT - for V2
+
+/** example response
```

Comment:
does swagger not generate these examples for us?

Reply:
There's a way of specifying it in-line as an [example value](https://github.com/swaggo/swag#example-value-of-struct), but just for readability sake for now I'll keep this one as a comment. It's moreso a reminder for me about how things will be shaped in the db schema